src/actions/chat.ts

```diff
@@ -25,6 +26,9 @@ export const getChats = () => async dispatch => {
       url: CHAT_URL,
       method: "get"
     });
+    _.map(chats, chat => {
```

Comment:
it's better
chats = _.map(chats, chat => ({...chat, chatColor: getRandomColor(chat._id)}))